CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Not released
+
+### Added
+
+- Add retries for tasks - [#92](https://github.com/ayrat555/fang/pull/92)
+- Add `remove_task_by_metadata` method to the Queue - [#90](https://github.com/ayrat555/fang/pull/92)
```

Comment:
```suggestion
- Add `remove_task_by_metadata` method to the Queue - [#90](https://github.com/ayrat555/fang/pull/90)
```

Reply:
these changes were already released so I removed this entry